plugins/modules/mso_backup.py

```diff
@@ -49,8 +49,9 @@
     description:
     - Use C(present) or C(absent) for adding or removing.
     - Use C(query) for listing an object or multiple objects.
+    - Use C(upload) for uploading backup.
```

Comment:
Probably want to add the other two (restore and download) too

Reply:
Will take care of this

Reply:
Does not seems to be fixed yet.